index.html

```diff
@@ -1678,6 +1678,45 @@ <h3>Bitstring Encoding</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Validity Periods</h3>
+
+      <p>
+The <a data-cite="?VC-DATA-MODEL-2.0#validity-period">validity period</a> of a
+status list is dependent on a variety of factors including:
```

Comment:
One reading of this is that the validity period of the status list can't just be read from the status list VC, which isn't true. I think this section is about how issuers might *choose* a validity period, not how it is expressed, definitively, in the status list VC. So I recommend a slight tweak here:

```suggestion
The <a data-cite="?VC-DATA-MODEL-2.0#validity-period">validity period</a> that
an issuer might choose to express in a status list is dependent on a variety of
factors including:
```